tests/index.tsx

```diff
@@ -473,17 +493,39 @@ test('element created by function with existing children and attributes', t => {
 });
 
 test('element created by function with combined children and attributes', t => {
-	const Icon = () => <i className="sweet">Gummy <span>bears</span></i>;
+	const Icon = ({children}: {children: Node[]}) => <i className="sweet">Gummy <span>bears</span>{children}</i>;
```

Comment:
Not required, but can you find the correct type here? I think it would be `Attributes`, `Props` or something like it. 
```suggestion
	const Icon = ({children}: Attributes) => <i className="sweet">Gummy <span>bears</span>{children}</i>;
```

Same goes for `Record<string, any>` used below

Reply:
I don't think it really matters, since React Component defined kinda like this:

```jsx
class Component<P = {}> {
    props: P;
    constructor(props: P) {
        this.props = props;
    }
}
```

So `props` can be of any type by definition. `Record<string, any>` suits it perfectly, imho

Reply:
That's a class; function components seem to use `React.FC`: https://www.carlrippon.com/Different-ways-to-strongly-type-function-component-props-with-typescript/

> `FC` includes the children prop type, so you don’t have to define it explicitly

Reply:
Speaking of which, it appears that the types might be broken for function components at the moment: https://github.com/sindresorhus/refined-github/pull/3904#discussion_r560722216

Could you try v5.0.0-2 to verify if it's just Refined GitHub or if it's `dom-chef`

Reply:
Possible example found here: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/41419